### PR TITLE
[Hotfix] Fix anchor scrolls in wiki and project settings page

### PR DESCRIPTION
**Pull request by caneruguz**
## Purpose
Fixes two separate but related problems, the scrolling of anchor tags in projects. 

## Changes
- Added a knockout binding to osfHelper called anchorScroll which basically takes over the scrolling of anchor links within an element but uses Knockout binding. 
- Cleaned up the project settings anchor link by taking out previous jquery fix which is now unnecessary and use html/css fix that was in place already (and adjust it to fit the current header).
- Fixes the affix view on the sidebar of the project settings page (The sidebar is supposed to scroll with the page so that the links are always visible and retain it's width when scrolled or when window is resized. 

## Side effects
This PR needs another one (#2041) to be merged before it gets merged so that the scroll element is correctly assigned in wiki pages.

### CODE REVIEW

**Comment by caneruguz** [comment]
Referencing #902 , as it may be related. Anchors anywhere on OSF should use this format: 
link:
```html
<li><a href="#selectAddonsAnchor">Select Add-ons</a></li>
```

just above the actual target add an anchor:
```html
<span id="configureAddonsAnchor" class="anchor"></span>
```

**Comment by caneruguz** [comment]
@Steve, this is ready to review. Adding @brianjgeiger.

File: website/static/js/osfHelpers.js [hunk 1]
```diff
@@ -295,6 +295,30 @@ ko.bindingHandlers.tooltip = {
...
 
+
+/**
+ * Takes over anchor scrolling and scrolls to anchor positions within elements 
+ * Example:
+ * <span data-bind="anchorScroll"></span>
+ */
+ko.bindingHandlers.anchorScroll = {
+    init: function(elem, valueAccessor) {
+        console.log( valueAccessor());
```
Comment (sloria) [inline website/static/js/osfHelpers.js]: :money_with_wings:

**Comment by caneruguz** [comment]
@Steve should I also send this as feature?

File: website/static/css/site.css [hunk 2]
```diff
@@ -966,6 +966,12 @@ h1.unavailable {
     font-style: italic;
 }
 
+.project-page .anchor {
+    margin-top: -100px;
```
Comment (sloria) [inline website/static/css/site.css]: Please add a code comment about why this is necessary

**Comment by sloria** [comment]
@caneruguz Can you resend this as a feature and get it up to date with develop?

**Pull request closed without merging**